test/src/tests-identities-attributes.ts

```diff
@@ -1188,19 +1291,26 @@ describe('identities and attributes', function() {
 
         fetchMock.resetHistory();
         // log back in with previous MPID, but with only a single UI, all UIs should be on batch
-        mockServer.respondWith(urls.login, [
-            200,
-            {},
-            JSON.stringify({ mpid: testMPID, is_logged_in: true }),
-        ]);
-
-        clock.tick(MILLISECONDS_IN_ONE_DAY_PLUS_ONE_SECOND);
 
+        fetchMockSuccess(urls.login, {
+            mpid: 'testMPID', is_logged_in: false
```

Comment:
```suggestion
            mpid: 'testMPID', is_logged_in: true
```

Reply:
see a few other spots like this, so i'll fix locally